src/controller/application.cpp

```diff
@@ -161,9 +164,14 @@ CommandWithArgumentsPtr Application::parseArgs()
         // https://bugs.chromium.org/p/chromium/issues/detail?id=354597#c2
         qDebug() << "Parent window handle is unused" << parser.value(parentWindow);
     }
+    if (parser.isSet(aboutArgument)) {
+        return std::make_unique<CommandWithArguments>(CommandType::ABOUT, QVariantMap());
+    }
+#ifndef Q_OS_LINUX
```

Comment:
Please add a comment that explains that this is needed to support Snap browsers.